go/teams/loader_ctx.go

```diff
@@ -44,6 +45,62 @@ func NewLoaderContextFromG(g *libkb.GlobalContext) LoaderContext {
 	}
 }
 
+type rawTeam struct {
+	ID             keybase1.TeamID                                        `json:"id"`
+	Name           keybase1.TeamName                                      `json:"name"`
+	Status         libkb.AppStatus                                        `json:"status"`
+	Chain          []json.RawMessage                                      `json:"chain"`
+	Box            *TeamBox                                               `json:"box"`
+	Prevs          map[keybase1.PerTeamKeyGeneration]prevKeySealedEncoded `json:"prevs"`
+	ReaderKeyMasks []keybase1.ReaderKeyMask                               `json:"reader_key_masks"`
+	// Whether the user is only being allowed to view the chain
+	// because they are a member of a descendent team.
+	SubteamReader    bool                               `json:"subteam_reader"`
+	Showcase         keybase1.TeamShowcase              `json:"showcase"`
+	LegacyTLFUpgrade []keybase1.TeamGetLegacyTLFUpgrade `json:"legacy_tlf_upgrade"`
+}
+
+func (r *rawTeam) GetAppStatus() *libkb.AppStatus {
+	return &r.Status
+}
+
+func (t *rawTeam) unpackLinks(ctx context.Context) ([]*chainLinkUnpacked, error) {
+	if t == nil {
+		return nil, nil
```

Comment:
This seems bad for a receiver. Caller should not call methods on nil objects. I consider it a go design bug that this is possible